$NetBSD: patch-src_CMakeLists.txt,v 1.1 2025/03/16 14:56:12 plunky Exp $

turn off debug output

include qtjsapi (library) and qcadjsapi (plugin) in build

--- src/CMakeLists.txt.orig	2025-02-20 12:25:22.000000000 +0000
+++ src/CMakeLists.txt
@@ -3,6 +3,8 @@ project(src VERSION 1.0 LANGUAGES CXX)
 
 include(../CMakeInclude.txt)
 
+add_compile_definitions(QT_DEBUG_NO_OUTPUT)
+
 add_subdirectory(3rdparty)
 add_subdirectory(core)
 add_subdirectory(spatialindex)
@@ -16,6 +18,8 @@ add_subdirectory(stemmer)
 add_subdirectory(customwidgets)
 add_subdirectory(run)
 add_subdirectory(scripts)
+add_subdirectory(qtjsapi)
+add_subdirectory(qcadjsapi)
 if(NOT BUILD_QT6)
     message("Qt 5")
     add_subdirectory(scripting)
